.github/workflows/blackhole-post-commit.yaml

```diff
@@ -24,6 +24,8 @@ jobs:
   build-docker-image:
     uses: ./.github/workflows/build-docker-artifact.yaml
     secrets: inherit
+    with:
+      os: "ubuntu-22.04-amd64"
```

Comment:
Derp.  Thanks for catching this.
How was it working?  Shouldn't it fail to find the image?

Reply:
It was failing a few branches but mysteriously working in a bunch of others... I have no idea how.

Reply:
Random guess: if some other workflow (eg: APC) builds the image and runs earlier, then it'll be in the registry.